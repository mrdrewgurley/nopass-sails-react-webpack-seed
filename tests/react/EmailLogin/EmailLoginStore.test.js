import Store from '../../../assets/app/EmailLoginComponent/stores/EmailLoginStore'

const state = Store().getState()

/**
 * React.EmailLoginStore
 * @desc Tests for the Email Login Store
 */
describe('EmailLoginStore.state', () => {
  it('should match the initial snapshot', () => {
    expect(state).toMatchSnapshot()
  })
})
