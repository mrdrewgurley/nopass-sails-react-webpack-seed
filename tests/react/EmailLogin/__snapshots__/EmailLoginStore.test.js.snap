// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailLoginStore.state should match the initial snapshot 1`] = `
Object {
  "EmailLoginButtonReducer": Object {
    "class": "enabled",
    "label": "SEND LOGIN EMAIL",
    "state": true,
  },
  "EmailLoginInputReducer": Object {
    "class": "enabled",
    "label": "SEND LOGIN EMAIL",
    "state": true,
  },
}
`;
